lib/safely_block.rb

```diff
@@ -31,6 +29,7 @@ def safely(options = {})
       sample = options[:sample]
       if sample ? rand < 1.0 / sample : true
         begin
+          e = prepend_with_tag(e, options[:tag])
```

Comment:
Also for backward compatibility, tagging should happen inside the `report_exception` method to leave the original exception unmodified.